ApplyToBecomeInternal/ApplyToBecomeInternal/Pages/Components/RecordDecisionPreview/default.cshtml

```diff
@@ -1,112 +1,113 @@
-                        }
-                        else
-                        {
-                            <strong class="govuk-tag govuk-tag--green" id="decision">@decisionName</strong>
-                        }
-
-                    </dd>
-
-                </div>
-                <div class="govuk-summary-list__row">
-                    <dt class="govuk-summary-list__key">
-                        Who made this decision
-                    </dt>
-                    <dd class="govuk-summary-list__value">
-
-                        @if (Model.Decision != null && Model.Decision.DecisionMadeBy != null)
-                        {
-                            <div id="decision-made-by">@Model.Decision.DecisionMadeBy.ToDescription()</div>
-                        }
-                        else
-                        {
-                            <span class="empty">Empty</span>
-                        }
-
-                    </dd>
-                </div>
-
-                @{
-                    var conditionsSet = Model.Decision.ApprovedConditionsSet.HasValue && Model.Decision.ApprovedConditionsSet.Value;
-                }
-
-                <div class="govuk-summary-list__row">
-                    <dt class="govuk-summary-list__key">
-                        Were any conditions set
-                    </dt>
-                    <dd class="govuk-summary-list__value" id="condition-set">
-                        @(conditionsSet ? "Yes" : "No")
-                    </dd>
-                </div>
-
-                @if (conditionsSet)
-                {
+@if (Model.Decision != null)
+{
+            <div class="govuk-grid-column-two-thirds">
+                <dl class="govuk-summary-list">
+
                     <div class="govuk-summary-list__row">
                         <dt class="govuk-summary-list__key">
-                            Details of the conditions
+                            Decision
                         </dt>
-                        <dd class="govuk-summary-list__value" id="condition-details">
-                            @Model.Decision.ApprovedConditionsDetails
+                        <dd class="govuk-summary-list__value">
+
+                    @{
+                        var decisionName = Model.Decision.GetDecisionAsFriendlyName();
+                    }
+                    @if (string.IsNullOrEmpty(decisionName))
+                    {
+                                    <span class="empty">Empty</span>
+                    }
+                    else
+                    {
+                                    <strong class="govuk-tag govuk-tag--green" id="decision">@decisionName</strong>
+                    }
+
+                        </dd>
+
+                    </div>
+                    <div class="govuk-summary-list__row">
+                        <dt class="govuk-summary-list__key">
+                            Who made this decision
```

Comment:
Missing a '?' ... ?

Reply:
This not a new code. The formatting has confused github.
I would've changed it, but the prototype says its right.
<img width="1343" alt="image" src="https://user-images.githubusercontent.com/109524605/184154294-ef69cefb-7c55-43db-b851-00604a82fc07.png">